docs/layouts/partials/navbar.html

```diff
@@ -22,7 +22,7 @@
 			</li>
 			{{ end }}
 			{{ if  .Site.Params.versions }}
-			<li class="nav-item dropdown d-none d-lg-block">
+			<li class="nav-item dropdown mr-2 mr-xl-3 mb-2 mb-lg-0">
```

Comment:
This is the change that actually makes the dropdown available on all resolutions (just like the other menu items)